src/rust/engine/src/nodes.rs

```diff
@@ -739,7 +550,7 @@ impl Snapshot {
     )
   }
 
-  fn store_files_content(context: &Context, item: &[FileContent]) -> Value {
+  pub fn store_files_content(context: &Context, item: &[FileContent]) -> Value {
```

Comment:
Move to `intrinsics.rs` instead?

Reply:
Thought about doing this, but it's one of a bunch of related associated functions on `Snapshot` and I don't think I want to move the whole type into `intrinsics.rs`, so I'd rather keep it defined where it is and just make the signature public.